toolchains/gcc-cxx11.cmake

```diff
@@ -0,0 +1,23 @@
+# Sample toolchain file for building with gcc compiler
+#
+# Typical usage:
+#    *) cmake -H. -B_build -DCMAKE_TOOLCHAIN_FILE="${PWD}/toolchains/gcc.cmake"
+
+# find compiler
+find_program(CMAKE_C_COMPILER gcc)
+find_program(CMAKE_CXX_COMPILER g++)
+if(NOT CMAKE_C_COMPILER)
+  message(FATAL_ERROR "gcc not found")
+endif()
+if(NOT CMAKE_CXX_COMPILER)
+  message(FATAL_ERROR "g++ not found")
+endif()
+
+# set compiler
+set(CMAKE_C_COMPILER   "${CMAKE_C_COMPILER}"   CACHE PATH "C compiler"   FORCE)
+set(CMAKE_CXX_COMPILER "${CMAKE_CXX_COMPILER}" CACHE PATH "C++ compiler" FORCE)
```

Comment:
Redundant. CMake will in the sysroot anyway. Setting `CMAKE_C_COMPILER` to `gcc` should already work.